Dockerfile

```diff
@@ -9,5 +9,5 @@ RUN SEMANTIC_VERSION=$semantic_version make build
 RUN mkdir -p /root/.cache/storageos
 
 # The storageos binary must be in the PATH for the examples in the docs to work.
-RUN ln -s bin/storageos /usr/local/bin/storageos
-ENTRYPOINT ["/storageos"]
+RUN ln -s /go/src/github.com/storageos/go-cli/bin/storageos /usr/local/bin/storageos
```

Comment:
We set the workdir to this path in line 3
So isn't the fix just removing the slack from the line below?

Reply:
Yep, you're right. I think the comment is misleading, the entrypoint just needs to be correct for the examples in the docs to work. I've fixed this